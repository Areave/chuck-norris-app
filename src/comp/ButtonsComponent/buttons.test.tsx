import React from 'react'
import Button from './button'
import renderer from 'react-test-renderer'
import {configure, shallow} from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

configure({adapter: new Adapter()})



describe('render <Button>', ()=> {

  it('<Button> rendered correctly', ()=> {
    const button = shallow(<Button label={'label'} onClick={()=>console.log('click!')}/>);
    expect(button).toMatchSnapshot()
  })
})


// describe('render <Button>', ()=> {

//   it('<Button> rendered correctly', ()=> {
//     const button = renderer.create(<Button label={'label'} onClick={()=>console.log('click!')}/>).toJSON();
//     expect(button).toMatchSnapshot()
//   })
// })