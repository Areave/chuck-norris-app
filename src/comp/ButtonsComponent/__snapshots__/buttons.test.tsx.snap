// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render <Button> <Button> rendered correctly 1`] = `ShallowWrapper {}`;
